src/styles/_catalog.scss

```
@@ -1,22 +1,40 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  gap: 48px;
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
   &__card:not(:nth-child(4n)) {
     margin-right: 48px;
   }
 }
+
+@media (max-width: 1023.9px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
```

please don't forget about nesting and please add this code to `.catalog`.
```suggestion
@media (max-width: 1023px) {
    grid-template-columns: repeat(3, 200px);
}
```

If you need any help or have questions - feel free to ask in chat or join to the next HTML/CSS Q&A session

---

please check all places